apps/demos/Demos/Chat/Overview/jQuery/index.js

```diff
@@ -0,0 +1,103 @@
+$(async () => {
+    const todayDate = new Date();
+    const yesterdayDate = (new Date()).setDate(todayDate.getDate() - 1);
+
+    const johnDoe = {
+        id: "c94c0e76-fb49-4b9b-8f07-9f93ed93b4f3",
+        name: "John Doe",
+    };
+    
+    const supportAgent = {
+        id: "d16d1a4c-5c67-4e20-b70e-2991c22747c3",
+        name: "Support Agent",
+        avatarUrl: "https://raw.githubusercontent.com/DevExpress/DevExtreme/refs/heads/24_2/apps/demos/images/employees/24.png",
+    };
+
+    let messages = [
+        {
+            timestamp: yesterdayDate,
+            author: supportAgent,
+            text: "Hello, John!\nHow can I assist you today?"
+        },
+        {
+            timestamp: yesterdayDate,
+            author: johnDoe,
+            text: "Hi, I'm having trouble accessing my account."
+        },
+        {
+            timestamp: yesterdayDate,
+            author: johnDoe,
+            text: "It says my password is incorrect."
+        },
+        {
+            timestamp: yesterdayDate,
+            author: supportAgent,
+            text: "I can help with that. Can you please confirm your email address for security purposes?"
+        },
+        {
+            timestamp: yesterdayDate,
+            author: johnDoe,
+            text: "Sure"
+        },
+        {
+            timestamp: yesterdayDate,
+            author: johnDoe,
+            text: "john.doe1357"
+        },
+        {
+            timestamp: yesterdayDate,
+            author: supportAgent,
+            text: "✅ Instructions to regain access have been sent to the email address you provided. "
+        },
+    ];
+
+    function onMessageEntered({ message }) {
+        messages = [...messages, message];
```

Comment:
I think we don't need it in jQuery demo while this array is used only for initial message specifying.
We can even rename it with `initialMessages`

Reply:
2679e0c3362d769e1ea3315578e64523e07689d8